src/components/ExpenseForm/ChooseSubcategory/ChooseSubcategory.jsx

```diff
@@ -0,0 +1,24 @@
+import React from 'react'
+
+import * as S from './../ChooseCategory/style';
+
+const ChooseSubcategory = params => {
+  function handleSubchangeCategory(e) {
+    params.onSubcategoryChange(e)
+  }
+
+  function setOptions () {
```

Comment:
Can this be moved to parent component and accordingly in `ChooseCategory` and merge them into one component?

Reply:
Merged into one file